packages/chat-core-zendesk/src/infra/ChatCoreZendeskImpl.ts

```diff
@@ -89,8 +89,23 @@ export class ChatCoreZendeskImpl {
    * with the conversation summary as the initial message.
    */
   private async setupSession(messageRsp: MessageResponse) {
+    const ticketFields: Record<string, unknown> = {};
+    try {
+      if (!!messageRsp.integrationDetails?.zendeskHandoff?.ticketFields) {
```

Comment:
the !! is redundant here; you generally only need that when using react rendering under specific circumstances (or if some parameter in TS explicitly demands a boolean)

Reply:
Makes sense! Fixed